examples/seq2seq/seq2seq.py

```diff
@@ -213,25 +219,26 @@ def main():
     target_words = {i: w for w, i in target_ids.items()}
     source_words = {i: w for w, i in source_ids.items()}
 
-    model = Seq2seq(3, len(source_ids), len(target_ids), args.unit)
+    model = Seq2seq(args.layer, len(source_ids), len(target_ids), args.unit)
     if args.gpu >= 0:
         chainer.cuda.get_device(args.gpu).use()
         model.to_gpu(args.gpu)
 
     optimizer = chainer.optimizers.Adam()
     optimizer.setup(model)
 
+    trigger = (4000, 'iteration')
     train_iter = chainer.iterators.SerialIterator(train_data, args.batchsize)
     updater = training.StandardUpdater(
         train_iter, optimizer, converter=convert, device=args.gpu)
-    trainer = training.Trainer(updater, (args.epoch, 'epoch'))
-    trainer.extend(extensions.LogReport(trigger=(200, 'iteration')),
-                   trigger=(200, 'iteration'))
+    trainer = training.Trainer(updater, (args.epoch, 'epoch'), out=args.out)
+    trainer.extend(extensions.LogReport(trigger=trigger),
+                   trigger=trigger)
     trainer.extend(extensions.PrintReport(
         ['epoch', 'iteration', 'main/loss', 'validation/main/loss',
          'main/perp', 'validation/main/perp', 'validation/main/bleu',
          'elapsed_time']),
-        trigger=(200, 'iteration'))
+        trigger=trigger)
```

Comment:
(This is just my feeling, so you can leave this.)
I think it is user-friendly to group extensions into two, "small step" and "large step".
"Small step" group includes PrintReport, LogReport and sample generation. They and important for checking progress at a low cost. And also, it is useful just for seeing running without problems.
"Large step" group includes BLEU calculation and snapshot serialization. They take a cost to some extent. BLEU calculation over test dataset seem to take much time. Serialization needs a large storage size without user's awareness (if frequently).
So, what do you think about preparing two different step (small and large) sizes for triggers?